bin/watch

```diff
@@ -13,17 +14,18 @@ function cleanup {
 cleanup pattern-library-gulp-watch
 cleanup pattern-library-php-generate
 cleanup pattern-library-php-watch
-docker-compose build
```

Comment:
For speed?

Reply:
Yes, even when everything is cached it would still add several seconds to the startup time of this script